redisson/src/main/java/org/redisson/client/protocol/RedisCommands.java

```diff
@@ -225,6 +226,17 @@
     RedisStrictCommand<Void> EVAL_VOID = new RedisStrictCommand<Void>("EVAL", new VoidReplayConvertor());
     RedisCommand<Object> EVAL_FIRST_LIST = new RedisCommand<Object>("EVAL", new ListFirstObjectDecoder());
     RedisCommand<List<Object>> EVAL_LIST = new RedisCommand<List<Object>>("EVAL", new ObjectListReplayDecoder<Object>());
+    RedisCommand<List<Integer>> EVAL_INT_LIST = new RedisCommand("EVAL", new ObjectListReplayDecoder<Integer>(), new Convertor<Integer>() {
```

Comment:
@mrniko couldn't get this to work with generics.. let me know what you think, in a nutshell it's a problem with multi decoders and converting the values before adding them to the array.

Reply:
in particular, here `org.redisson.client.handler.CommandDecoder#decode(io.netty.buffer.ByteBuf, org.redisson.client.protocol.CommandData<java.lang.Object,java.lang.Object>, java.util.List<java.lang.Object>, io.netty.channel.Channel, boolean, java.util.List<org.redisson.client.protocol.CommandData<?,?>>)` where code is `:`, reads a long, then adds it to the parts `List<Object>`..

Reply:
done.